components/service/fxa/.gitignore

```diff
@@ -0,0 +1,3 @@
+# jniLibs
+*.so
+!libjnidispatch.so
```

Comment:
libjnidispatch.so is something we commit to the repo?

Reply:
Yeah, we can't get that easily from anywhere right now. I will file a follow-up to find a solution.